README.md

```diff
@@ -43,31 +43,36 @@ let vstsBuild: ba.IBuildApi = connection.getBuildApi();
 
 #### Available clients
 
+These clients are available in the 6.x API:
```

Comment:
These clients are available in the new 6.x preview API (`npm install vso-node-api@preview`)